### DISCUSSION TITLE
What is an example of an `app` in the Endpoint Testing docs?

### DISCUSSION THREAD

**Discussion by corysimmons** [avajs/ava#2692] [Q&A] [answered]
https://github.com/avajs/ava/blob/main/docs/recipes/endpoint-testing.md

Would this just be something like:

```js
// ../app.js
module.exports = some_express_app
```

**Comment by novemberborn** [comment, marked as answer]
The example uses the built-in `http` module, so it would be a `requestListener` along the lines of https://nodejs.org/docs/latest/api/http.html#http_http_createserver_options_requestlistener.
